frontend/components/topics/Tag.tsx

```diff
@@ -0,0 +1,31 @@
+import { Topic } from "@/interfaces";
+import Chip from "@material-ui/core/Chip";
+import { makeStyles } from "@material-ui/styles";
+import Link from "next/link";
+import { FC } from "react";
+
+const useStyles = makeStyles({
+  tag: {
+    backgroundColor: "rgba(255, 255, 153, 0.8)",
+    fontSize: "0.7rem",
+  },
+});
+
+interface TagProps {
+  topic: Topic;
+}
+
+const Tag: FC<TagProps> = ({ topic }: TagProps) => {
```

Comment:
Do we prefer the name `Tag`? Or could we rename this component to `Chip` or `ChipLink` (or something else that might better convey what the component is).

Reply:
I think I'd rather not name it `Chip` since I intended for the component to be used specifically for topic tags (not as a themed version of MUI's chip or something like that)—but, I don't have a strong opinion.